eclass/dist-kernel-utils.eclass

```diff
@@ -104,12 +104,23 @@ dist-kernel_install_kernel() {
 	if [[ ${magic} == MZ ]]; then
 		einfo "Combined UEFI kernel+initramfs executable found"
 		# install the combined executable in place of kernel
-		image=${initrd}.uefi
+		image=${initrd}.efi
 		mv "${initrd}" "${image}" || die
-		# put an empty file in place of initrd.  installing a duplicate
-		# file would waste disk space, and removing it entirely provokes
-		# kernel-install to regenerate it via dracut.
-		> "${initrd}"
+		# We moved the generated initrd, prevent dracut from running again
+		local plugins=()
+		for file in "${EROOT}"/usr/lib/kernel/install.d/*; do
```

Comment:
```suggestion
		for file in "${EROOT}"/usr/lib/kernel/install.d/*.install; do
```

Then you don't need the extra check below.

Reply:
Oops, yes of course, clearly I wasn't fully awake when I wrote this :P